functions/src/express_configs/express_portal.ts

```diff
@@ -37,7 +37,7 @@ app.use(Sentry.Handlers.tracingHandler());
 
 app.use(
   cors({
-    origin: ["https://app.acmutd.co", "http://localhost:3000", "https://portal.acmutd.co"],
+    origin: [environment.URL_ORIGINS || ""],
```

Comment:
Why do we have `|| ""`?

Reply:
I'm not sure what the best practice or elegant solution to get around `Type 'string | undefined' is not assignable to type 'string | RegExp'`

Reply:
Try [type assertions](https://basarat.gitbook.io/typescript/type-system/type-assertion). This might work.
```suggestion
    origin: [environment.URL_ORIGINS as string],
```

Reply:
Is it possible to change the doppler config to return an array instead of multiple comma separated strings? I think the code would look more readable if we could have it be something like this:

`origin: environment.URL_ORIGINS` instead of enclosing it within the array brackets []

Reply:
Yes that works. 
```suggestion
    origin: environment.URL_ORIGINS,
```

Reply:
https://github.com/w9jds/firebase-action since we already have doppler integration into github, I think we can use github actions for CI/CD to firebase.

Reply:
https://github.com/marketplace/actions/install-doppler-cli This could be viable.